modules/osdk-bundle-operator.adoc

```diff
@@ -11,6 +12,10 @@ endif::[]
 ifeval::["{context}" == "osdk-working-bundle-images"]
 :golang:
 endif::[]
+ifeval::["{context}" == "osdk-java-tutorial"]
```

Comment:
Did you mean to add a prereq for Java-based Operators? If not, you don't need these if statements.

Reply:
Good point - removing the ifevals.